docs/02-app/01-building-your-application/01-routing/10-parallel-routes.mdx

```diff
@@ -122,7 +122,7 @@ export default function Layout({ auth }: { auth: React.ReactNode }) {
 import { useSelectedLayoutSegment } from 'next/navigation'
 
 export default function Layout({ auth }) {
-  const loginSegments = useSelectedLayoutSegment('auth')
+  const loginSegment = useSelectedLayoutSegment('auth')
```

Comment:
Looks like this needs updating in the text snippet below as well

Reply:
thanks, fixed in https://github.com/vercel/next.js/pull/64029/commits/7714d988cfdd96b44071da42b8b509bd154c0f45